pkg/platform/local/platform.go

```diff
@@ -81,62 +81,95 @@ func (p *Platform) DeployFunction(deployOptions *platform.DeployOptions) (*platf
 	})
 }
 
-// GetFunctions will return deployed functions
-func (p *Platform) GetFunctions(getOptions *platform.GetOptions) ([]platform.Function, error) {
-	getContainerOptions := &dockerclient.GetContainerOptions{
-		Labels: map[string]string{
-			"nuclio-platform":  "local",
-			"nuclio-namespace": getOptions.Namespace,
-		},
-	}
-
-	// if we need to get only one function, specify its function name
-	if getOptions.Name != "" {
-		getContainerOptions.Labels["nuclio-function-name"] = getOptions.Name
-	}
-
-	containersInfo, err := p.dockerClient.GetContainers(getContainerOptions)
-
-	if err != nil {
-		return nil, errors.Wrap(err, "Failed to get containers")
+// Unduplicate takes array of platform.MatchCriteria and returns the array unduplicated
+func (p *Platform) Unduplicate(matchCriteria []platform.MatchCriteria) []platform.MatchCriteria {
```

Comment:
- I don't think this should be here. You get a list of string arguments in cobra's `command/get.go` - why not remove duplicates from it right then?

- Variable naming and comments are vital to algorithmic functions like this. I have rewritten the function using terms that are a bit more understandable and added comments throughout explaining the duplicate removal process:
```go
func removeDuplicates(args []string) []string {
	uniqueArgs := []string{}
	for _, arg := range args {

		// assume the argument hasn't been found in uniqueArgs
		argExists := false

		for _, uniqueArg := range uniqueArgs {
			if arg == uniqueArg {

				// mark that we found it and break out
				argExists = true
				break
			}
		}

		if !argExists {

			// if we haven't found the argument in uniqueArgs, add it here
			uniqueArgs = append(uniqueArgs, arg)
		}
	}

	return uniqueArgs
}
```